Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -170,7 +170,11 @@ file that was distributed with this source code.
             {{- widget|raw -}}
             {%- if label is not same as(false) -%}
                 <span class="control-label__text">
-                    {{- label|trans({}, translation_domain) -}}
+                    {% if translation_domain is same as(false) %}
```

Comment:
Could you have concrete example and results of this not working case?

Reply:
If you don't want to translate the choice of an expanded choice type you can set choice_translation_domain = false and it will not translate them anymore. Specially if they are create by User input like UserGroup

Currently you can't do that, and it will always try to translate them

Reply:
I think this is okay for this case. This is a common way for translation inside symfony forms @Soullivaneuh